doc/api/cli.md

```diff
@@ -129,6 +129,20 @@ added: v2.1.0
 Prints a stack trace whenever synchronous I/O is detected after the first turn
 of the event loop.
 
+### `--enable-tracing`
+<!-- YAML
+added: v7.1.0
+-->
+
+Enables the collection of trace event tracing information.
+
+### `--enabled-categories=list`
+<!-- YAML
+added: v7.1.0
+-->
+
+A comma separated list of categories that should be traced when trace event
```

Comment:
For now the only available categories are inside V8. No special action is required to add a custom category. Adding a trace event such as `TRACE_EVENT_BEGIN0("MY_CATEGORY", "SomethingCostly")` will create the category `"MY_CATEGORY"` if it doesn't exist. As per https://github.com/nodejs/node/pull/9304#discussion_r85846385 the config file has been removed. I've updated the PR description to reflect this.